lib/webpacker/digests.rb

```diff
@@ -1,5 +1,5 @@
 # Singleton registry for accessing the digested filenames computed by Webpack in production mode.
-# This allows javascript_pack_tag to take a reference to, say, "calendar.js" and turn it into 
+# This allows javascript_pack_tag to take a reference to, say, "calendar.js" and turn it into
 # "calendar-1016838bab065ae1e314.js". These digested filenames are what enables you to long-term
 # cache things in production.
 class Webpacker::Digests
```

Comment:
@gauravtiwari We need to fix the errors in this file so they are crystal clear.

If somebody starts the rails server without first starting, and having the compilation complete, we won't have a digest file.

The typical case is to use a profile to start both the rails server and the dev server in watch mode for generating static files. The rails server will start first, and then there will be no manifest.

Here is an example:
https://github.com/shakacode/react_on_rails/blob/master/spec/dummy/Procfile.static

```
# Run Rails without hot reloading (static assets).
rails: REACT_ON_RAILS_ENV= rails s -b 0.0.0.0

# Build client assets, watching for changes.
rails-client-assets: sh -c 'yarn run build:dev:client'

# Build server assets, watching for changes. Remove if not server rendering.
rails-server-assets: sh -c 'yarn run build:dev:server'
```